flow/view_holder.cc

```diff
@@ -16,35 +16,7 @@ using ViewHolderBindings =
 FML_THREAD_LOCAL fml::ThreadLocalUniquePtr<ViewHolderBindings>
     tls_view_holder_bindings;
 
-fuchsia::ui::gfx::ViewProperties ToViewProperties(float width,
-                                                  float height,
-                                                  float insetTop,
-                                                  float insetRight,
-                                                  float insetBottom,
-                                                  float insetLeft,
-                                                  bool focusable) {
-  return fuchsia::ui::gfx::ViewProperties({
-      .bounding_box = fuchsia::ui::gfx::BoundingBox({
-          .min = fuchsia::ui::gfx::vec3({
-              .x = 0.f,
-              .y = 0.f,
-              .z = -1000.f,
-          }),
-          .max = fuchsia::ui::gfx::vec3({.x = width, .y = height, .z = 0.f}),
-      }),
-      .inset_from_min = fuchsia::ui::gfx::vec3({
-          .x = insetLeft,
-          .y = insetTop,
-          .z = 0.f,
-      }),
-      .inset_from_max = fuchsia::ui::gfx::vec3({
-          .x = insetRight,
-          .y = insetBottom,
-          .z = 0.f,
-      }),
-      .focus_change = focusable,
-  });
-}
+constexpr float kArbitraryLargeDepth = -1000.f;
```

Comment:
Can you add a comment about why this value was chosen so that we can update it later if needed?

Reply:
Done